docs/configuration.asciidoc

```diff
@@ -142,6 +142,34 @@ Below is a sample `Web.config` configuration file for a ASP.NET application.
 </configuration>
 ----
 
+Additionally on ASP.NET you can also implement your own configuration reader. For this you need to implement the `IConfigurationReader` interface from the `Elastic.Apm.Config` namespace.
+
+Once you have your `IConfiguration` implementation you can use the <<config-full-framework-configuration-reader-type, FullFrameworkConfigurationReaderType>> setting.
+
+[float]
+[[config-full-framework-configuration-reader-type]]
+==== `FullFrameworkConfigurationReaderType`
+
```

Comment:
I would add something like this to the top

```suggestion

NOTE: Use of this setting will disable {apm-app-ref}/agent-configuration.html[Central configuration].

```

Reply:
As noted [here](https://github.com/elastic/apm-agent-dotnet/pull/920#discussion_r464632697) I was originally wrong, so this'll work with central config.